OpenDreamServer/Dream/Objects/MetaObjects/DreamMetaObjectWorld.cs

```diff
@@ -59,7 +59,11 @@ class DreamMetaObjectWorld : DreamMetaObjectRoot {
             } else if (variableName == "maxz") {
                 return new DreamValue(Program.DreamMap.Levels.Count);
             } else if (variableName == "address") {
-                return DreamValue.Null;
+                return new(Program.DreamServer.Address.ToString());
+            } else if (variableName == "port") {
+                return new(Program.DreamServer.Port.ToString());
+            } else if (variableName == "url") {
+                return new("byond://" + Program.DreamServer.Address + ":" + Program.DreamServer.Port);
```

Comment:
I'm hesitant to have this return a `byond://` url

Reply:
fair, was just replicating existing behavior - we should probably specify our own thing